app/assets/stylesheets/dm/pages/_partials/_header.scss

```diff
@@ -295,14 +299,18 @@
 
   a {
     background-image: image-url('crisis-line-logo.png');
-    background-color: color($theme-color-primary-lighter) !important;
-    color: color($theme-color-base-ink)!important;
+    background-color: #801C11; 
+    color: #FFF;
     background-repeat: no-repeat;
     line-height: 15px !important;
     background-size: 27px 17.5px;
     background-position: 16px center;
     padding: 8px 16px 8px 56px;
 
+    &:active, &:focus, &:hover  {
+      background-color: #801C11; 
+    }
+
```

Comment:
It looks like lines `291`-`313` are duplicates of `256`-`289`. I think we can just delete `291`-`313`. Not sure why that's there lol

Reply:
Alrighty! I pushed up a new commit that deletes these. The one small difference I noticed is the former uses `right:24px` and the latter uses: `right: 3rem`. I went with `right: 3rem` to match prod.